packages/nimble-components/src/wafer-map/specs/features/rendering.md

```diff
@@ -82,7 +82,49 @@ The POC is found in this branch [Worker Rendering POC](https://github.com/ni/nim
 
 ### Data Structure and Interface
 
-We have two possible solutions for representing the data in the memory. They will be decided with a spec update. The fist one is an in-house solution:
+The best solution to solve the API of the wafermap is to use parts of both of the proposed methods mentioned in the next section. This means using Apache Arrow as the wafer component API and Typed Arrays for their iterating performance and transferability to worker threads.
+
+The Public API will be the following:
+
+```TS
+public diesTable: Table<{
```

Comment:
To make it even more clear where the `Table` type is coming from you could add something like this (feel free to change the import path if it's not right):
```suggestion
import { Table } from 'apache-arrow';

public diesTable: Table<{
```